### PR TITLE
[fix] #1783: Fixed torii benchmark.

### LINKED ISSUE #1783: Restore benchmarks
Currently, benchmarks don't work. 

# MWE: 
run 
```cargo bench -p iroha_client```
## Expected
Runs benchmark
## Actual benchmark fails with 
```
invalid url or method: Error(InvalidBaseUrl)
```

### PR DESCRIPTION
**Pull request by appetrosyan**
Signed-off-by: Aleksandr <a-p-petrosyan@yandex.ru>

### Description of the Change
Fixes the benchmarks.

### Issue
Closes #1783 

### Benefits
Benchmarks work

### Possible Drawbacks

None

### Usage Examples or Tests *[optional]*

```
cd client
cargo bench
```

### CODE REVIEW

File: client/benches/torii.rs [hunk 1]
```diff
@@ -59,16 +58,27 @@ fn query_requests(criterion: &mut Criterion) {
         IdBox::AssetId(AssetId::new(asset_definition_id, account_id.clone())),
     );
     let mut client_config = iroha_client::samples::get_client_config(&get_key_pair());
+
+    // TODO: We should definitely turn this into type-checked state. `String` is a terrible bodge
```
Comment (mversic) [inline client/benches/torii.rs:62]: would you mid opening an issue for this?

**Review by mversic — DISMISSED** [review]

File: client/benches/torii.rs (hunk 1 above)
Comment (appetrosyan) [reply client/benches/torii.rs:62]: #1786

File: client/benches/torii.rs [hunk 2]
```diff
@@ -59,16 +58,27 @@ fn query_requests(criterion: &mut Criterion) {
...
     client_config.torii_api_url = peer.api_address.clone();
+    if !client_config.torii_api_url.starts_with("http://") {
+        client_config.torii_api_url = format!("http://{}", client_config.torii_api_url);
+    }
+    if !client_config.torii_telemetry_url.starts_with("http://") {
+        client_config.torii_telemetry_url = format!("http://{}", client_config.torii_telemetry_url);
+    }
     let mut iroha_client = Client::new(&client_config);
+    thread::sleep(std::time::Duration::from_millis(5000));
```
[the remark this answers is not in the document]
Comment (appetrosyan) [reply client/benches/torii.rs:71]: It almost always times out. I created #1789 to investigate that separately

**Review by mversic — APPROVED** [review]

File: client/tests/million_accounts.rs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        true,
+        generate_genesis(1000000),
+        &configuration.genesis,
+        configuration.sumeragi.max_instruction_number,
+    )
+    .expect("genesis creation failed");
+
+    rt.block_on(peer.start_with_config(genesis, configuration));
+
+    thread::sleep(std::time::Duration::from_millis(50000));
```
[the remark this answers is not in the document]
Comment (s8sato) [reply client/tests/million_accounts.rs:52]: What is this sleep for?

**Review by mversic — DISMISSED** [review]

File: client/tests/million_accounts.rs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    let rt = Runtime::test();
+    let genesis = GenesisNetwork::from_configuration(
+        true,
+        generate_genesis(1000000),
+        &configuration.genesis,
+        configuration.sumeragi.max_instruction_number,
+    )
+    .expect("genesis creation failed");
+
+    rt.block_on(peer.start_with_config(genesis, configuration));
```
Comment (s8sato) [inline client/tests/million_accounts.rs:50]: This is just a thread spawning and checks nothing.
Need to confirm whether the accounts created

Comment (appetrosyan) [reply client/tests/million_accounts.rs:57]: That would take an incredibly long time, and isn't exactly in-line with what the test is for. I'll add an explanation into the comments.

**Review by s8sato — APPROVED** [review]

**Review by mversic — APPROVED** [review]

**Pull request merged**